setup.cfg

```diff
@@ -61,7 +61,7 @@ virtualenv.activate =
     batch = virtualenv.activation.batch:BatchActivator
     fish = virtualenv.activation.fish:FishActivator
     powershell = virtualenv.activation.powershell:PowerShellActivator
-    python = virtualenv.activation.python:PythonActivator
```

Comment:
I'm getting an error that might be related to this merge.  Is the change expected?

`AttributeError: module 'virtualenv.activation.python' has no attribute 'PestythonActivator'`

Reply:
This just broke all of my production build pipelines as well.

Reply:
I have to be honest, have no idea how this change made it in. Late Friday releases are always dangerous but wanted to get out a fix release to address peoples issues due to https://github.com/pypa/virtualenv/pull/1846 🤦 🤷‍♂️

Reply:
It happens. Thanks for the quick fix! @gaborbernat